### PR TITLE
fix: Clarity and copy changes on enforced txs and gas limit and zktrie

### PR DESCRIPTION
**Pull request by dghelm**
- adds clearer expectations and numbers around `gas limit` (along with mentioning refund) but keeps testnet message passing number
- rewords enforced tx callout to make clear this isn't enabled
- fixes typo from #119

### CODE REVIEW

File: src/content/docs/en/developers/guides/scroll-messenger-cross-chain-interaction.mdx [hunk 1]
```diff
@@ -84,8 +84,8 @@ We pass the message by executing `executeFunctionCrosschain` and passing the fol
 - `value`: In this case, it is `0` because the `setGreeting`is not payable.
 - `greeting`: This is the parameter that will be sent through the message. Try passing `“This message was crosschain!”`
 - `gasLimit`:
-  - If you are sending the message from L1 to L2, around `5000` gas limit should be more than enough.
-  - If you are sending the message from L2 to L1, pass `0`, as the transaction be completed by executing an additional transaction on L1.
+  - If you are sending the message from L1 to L2, around `5000` gas limit should be more than enough. That said, if you set this too high, any unused portion of the fee will be refunded.
```
Comment (isabellewei) [inline src/content/docs/en/developers/guides/scroll-messenger-cross-chain-interaction.mdx]: The gas limit is not refunded. The `gasLimit` * `baseFee` is the fee that is charged, and if the `msg.value` is higher than this fee, the remainder of the `msg.value` is refunded. Conversely, if the `msg.value` is lower than this fee, the tx will revert

File: src/content/docs/en/developers/l1-and-l2-bridging/eth-and-erc20-token-bridge.mdx [hunk 2]
```diff
@@ -28,10 +28,9 @@ When bridging ERC20 tokens, you don’t have to worry about selecting the right
 All Gateway contracts will form the message and send it to the `L1ScrollMessenger` which can send arbitrary messages to L2. The `L1ScrollMessenger` passes the message to the `L1MessageQueue`. Any user can send messages directly to the Messenger to execute arbitrary data on L2. This means they can execute any function on L2 from a transaction made on L1 via the bridge. Although an application could directly pass messages to existing token contracts, the Gateway abstracts the specifics and simplifies making transfers and calls.
 
 <Aside type="tip" title="">
-  Users can also bypass the `L1ScrollMessenger` and send messages directly to the `L1MessageQueue`. If a message is sent
+  In future upgrades, users will be able to bypass the `L1ScrollMessenger` and send messages directly to the `L1MessageQueue`. If a message is sent
```
Comment (isabellewei) [inline src/content/docs/en/developers/l1-and-l2-bridging/eth-and-erc20-token-bridge.mdx:31]: Should we also make this clear in the Enforced Tx page in the docs?

**Review by isabellewei — CHANGES_REQUESTED** [review]
Sorry I think I didn't explain very clearly, gas limit is NOT refunded, but any extra `msg.value` will be and a too low `msg.value` will cause the tx to revert, which is why `msg.value` should always err on being set very high. 

The gas limit does affect the fee that is deducted from the user, I think we can mention somewhere that they can use `eth_estimateGas` to get a more accurate real-time estimate for what to set it

File: src/content/docs/en/developers/guides/scroll-messenger-cross-chain-interaction.mdx (hunk 1 above)
Comment (dghelm) [reply src/content/docs/en/developers/guides/scroll-messenger-cross-chain-interaction.mdx]: Fixed!

**Pull request merged**